lib/signal-list-view.js

```diff
@@ -1,132 +1,126 @@
-"use babel";
+/* @flow */
 
-import { SelectListView } from "atom-space-pen-views";
+import SelectListView from "atom-select-list";
 import _ from "lodash";
 
 import WSKernel from "./ws-kernel";
 import kernelManager from "./kernel-manager";
 import { log } from "./utils";
 import store from "./store";
 
-// View to display a list of grammars to apply to the current editor.
-export default class SignalListView extends SelectListView {
-  initialize() {
-    super.initialize(...arguments);
-
+export default class SignalListView {
+  basicCommands: Array<Object>;
+  editor: ?atom$TextEditor;
+  // onConfirmed: ?(KernelCommand) => void;
```

Comment:
Looking at the error message you sent me via slack the syntax should be
```js
onConfirmed: ?(command: KernelCommand) => void;
```
Similar to [this line](https://github.com/nteract/hydrogen/blob/master/lib/kernel-picker.js#L10)